src/js/main.js

```diff
@@ -0,0 +1,44 @@
+let fourToFour = document.getElementById('fourToFour');
+let thirtyToThirty = document.getElementById('thirtyToThirty');
+let image = document.getElementById('image');
+
+fourToFour.onclick = function() {
+    draw(1)
+};
+thirtyToThirty.onclick = function() {
+    draw(2)
+};
+image.onclick = function() {
+    draw(3)
+};
+
+function draw(stage) {
+    var ctx = document.getElementById('canvas').getContext('2d');
+    if (stage == 1) {
+        for (var i = 0; i < data.length; i++) {
+            for (var j = 0; j < data.length; j++) {
+                ctx.fillStyle = "#" + data[i][j];
+                ctx.fillRect(j * 128, i * 128, 128, 128);
```

Comment:
Please, try to avoid magic numbers.

https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544